docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -231,6 +231,50 @@ docker run \
 
 The default Intel Quote Provider config is available in [Intel SGX Github repository](https://github.com/intel/SGXDataCenterAttestationPrimitives/blob/master/QuoteGeneration/qcnl/linux/sgx_default_qcnl.conf).
 
+## Migrate from EPID Attestation to DCAP Attestation
+EPID attestation will be discontinued in 2025 and will no longer be available on
+any processors. All nodes using EPID attestation should migrate to DCAP
+Attestation.
+
+:::info
+
+Compute node operators, please hold for further instructions. We will coordinate
+the migration in phases to ensure that during the migration the network remains
+operational.
+
+:::
+
+:::info
+
+In case you are running AESM service on Docker please see [these instructions].
+
+:::
+
+To migrate to the DCAP Attestation run the following commands:
+1. See if your system [supports DCAP attestation].
+2. Stop your TEE nodes (Sapphire and Cipher).
```

Comment:
```suggestion
2. Gracefully shutdown your TEE nodes (Sapphire and Cipher).
```

This should probably link to: https://docs.oasis.io/node/run-your-node/maintenance/shutting-down-a-node/
So that nodes are greacefuly shutdown to avoid runtime disruption.